easybuild/easyconfigs/b/Bazel/Bazel-3.7.2-GCCcore-11.2.0.eb

```
@@ -0,0 +1,45 @@
+name = 'Bazel'
+version = '3.7.2'
+
+homepage = 'https://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably.
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = [
+    'https://github.com/bazelbuild/%(namelower)s/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
+patches = [
+    '%(name)s-3.4.1-fix-grpc-protoc.patch',
+    'java15.patch',
+    '%(name)s-3.7.1_fix-protobuf-env.patch',
+    'gcc11-bazel-3.7.2.patch',
+    'gcc11-2-bazel-3.7.2.patch',
+
+]
+checksums = [
+    # bazel-3.7.2-dist.zip
+    'de255bb42163a915312df9f4b86e5b874b46d9e8d4b72604b5123c3a845ed9b1',
+    # Bazel-3.4.1-fix-grpc-protoc.patch
+    'f87ad8ad6922fd9c974381ea22b7b0e6502ccad5e532145f179b80d5599e24ac',
+    '28f394f561824abf562556877483bc443dfd3ee0c7cba48a56e22670258669e7',  # java15.patch
+    # Bazel-3.7.1_fix-protobuf-env.patch
+    '8706ecc99b658e0a96c38dc2c23e44da35059b85f308602aac76a6d6680376e7',
+    # gcc11-bazel-3.7.2.patch
+    '32dc4abdc45bfcf70e9d93a66150b89646c801f8704930f0b713038366e40d40',
+    # gcc11-2-bazel-3.7.2.patch
+    '3160caff050330e33df1f7afb649e8303bf4af969ef7722639bd0f3c3ae036b4',
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('Python', '3.9.6'),
+    ('Zip', '3.0'),
+]
+
+prebuildopts = "export BAZEL_LINKOPTS=-static-libstdc++:-static-libgcc BAZEL_LINKLIBS=-l%:libstdc++.a:-lm && "
+configopts = "--host_jvm_args=--illegal-access=debug"
+dependencies = [('Java', '13', '', True)]
```

@surak We might want to use either Java 15 or 17 once the corresponding PR is merged: https://github.com/easybuilders/easybuild-easyconfigs/pull/14666 https://github.com/easybuilders/easybuild-easyconfigs/pull/14667
```suggestion
dependencies = [('Java', '15', '', True)]
```